python/py-multidict/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-multidict
+version             4.3.1
+categories-append   devel
+platforms           darwin
+license             Apache-2
+
+python.versions     34 35 36
+
+maintainers         {ipglider.org:miguel @ipglider} openmaintainer
+
+description         Multidict implementation.
+long_description    Multidict is dict-like collection of key-value pairs where \
+                    key might be occurred more than once in the container.
```

Comment:
Grammar isn't right. Perhaps "where a key might occur more than once in the container"?

Reply:
I just copied the description from the original documentation. I will change it.

Reply:
Do a pull request upstream, too. :)